saagieapi/pipelines/gql_queries.py

```diff
@@ -8,29 +8,38 @@
         }
     }
 }
-  """
+"""
 
 GQL_LIST_PIPELINES_FOR_PROJECT = """
-query projectPipelinesQuery($projectId: UUID!, $instancesLimit: Int, $versionsLimit: Int, $versionsOnlyCurrent: Boolean) {
+query projectPipelinesQuery($projectId: UUID!, 
+							$instancesLimit: Int, 
+                            $versionsLimit: Int, 
+                            $versionsOnlyCurrent: Boolean) {
     project(id: $projectId){
         pipelines{
             id
             name
             description
             alerting{
-              emails
-              loginEmails{
-                login
-                email
-              }
-              statusList
+                emails
+                loginEmails{
+                    login
+                    email
+                }
+                statusList
             }
             pipelineInstanceCount
             instances(limit: $instancesLimit){
-              id
-              status
-              startTime
-              endTime
+                id
+                status
+                startTime
+                endTime
+                runWithExecutionVariables
```

Comment:
The foc of the section `Listing pipelines for a project` should be updated (field `runWithExecutionVariables`, `initialExecutionVariables` in `instances` )